client/src/commons/PhoneNumberTextField/PhoneNumberTextField.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {TextField, StandardTextFieldProps} from '@material-ui/core';
+
+import CircleTextField from 'commons/CircleTextField/CircleTextField';
+import Validator from 'Utils/Validations/Validator';
+
+const PHONE_LABEL = 'טלפון:';
+const NOT_PHONE_ERROR = 'שגיאה: מספר שהוזן אינו תקין';
+
+const PhoneNumberTextField: React.FC<Props> = (props: Props): JSX.Element => {
+    const [isError, setIsError] = React.useState<boolean>(false);
+
+    const {id, value, onChange, testId, ...rest } = props;
+
+    return (
+        <CircleTextField
+            {...rest}
+            error={isError}
+            onBlur={() => setIsError(value && !Validator.phoneValidation(value as string))}
+            onFocus={() => setIsError(false)}
+            label={isError && NOT_PHONE_ERROR}
+            test-id={testId}
+            id={id}
+            placeholder={PHONE_LABEL}
```

Comment:
Remove that and send it as a prop, because some of the forms don’t have that and others won’t necessarily want that text